make_mt_IHME.py

```diff
@@ -0,0 +1,254 @@
+                ICD_non_explode = [ICD for ICD in ICD_fields if ICD != explode_icd]
+                mt_explICD = mt_fc.drop(*ICD_non_explode)
+                mt_fin = mt_explICD.explode_rows(mt_explICD[explode_icd])
+            else:
+                raise ValueError('If exploding the final table by one of the ICD columns,' +
+                                 ' the provided column must actually contain ICD codes.')
+        else:
+            mt_fin = mt_fc
+        return mt_fin.entries()
+    else:
+        raise ValueError('If trimming MatrixTable by an age group, provided age groups ' +
+                         'must be present in the MatrixTable column fields.')
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--input', type=str, help='Location of the table containing the data for merging. Expects this to contain data for a single year.')
+parser.add_argument('--output', type=str, help='Output MatrixTable.')
+parser.add_argument('--year', type=int, help='Year from which data was obtained.', default=2019)
+parser.add_argument('--col-key', type=str, help='The column that will be the column field key.', default='age_name')
+parser.add_argument('--col-other', type=str, help='Other columns to include as column fields, comma delimited.', default='age')
+parser.add_argument('--row-key', type=str, help='The column that will be the column field key.', default='cause_name')
+parser.add_argument('--row-other', type=str, help='Other columns to include as column fields, comma delimited. If blank, will include all other columns. Will convert any row fields matching ICD to an array by splitting on ", ".', default='')
+parser.add_argument('--entry-substr', type=str, help='Substrings, column delimited, to define an entry column. Uses OR between provided substraings.', default='val,upper,lower')
+parser.add_argument('--entry-segment-order', type=str, help='Entry columns are those that contain the substrings in --entry-subsr. They are period-delimited to represent hail structs. This comma-delimited string defines nesting order.', default='3,2,1')
+parser.add_argument('--output-table', type=str, help='Output a derivative Hail Table to a directory. If not provided, will not output. If provided, expects --filter-specific-column to select a column for the final table.', default='')
+parser.add_argument('--filter-specific-column', type=str, help='Filter the MatrixTable to a specific column for conversion to a Table. Only considered if --output-table is provided. Note that the column "Age-standardized" only contains rates.', default='Age-standardized')
+parser.add_argument('--explode-icd', type=str, help='Explode an ICD column. If not provided, will not explode. If provided, will remove all other ICD columns. Only considered if --output-table is provided.', default='')
+
+
+if __name__ == '__main__':
+    args = parser.parse_args()
+    
+    # Import
+    ht = hl.import_table(args.input, force=True, impute=True)
```

Comment:
If you're making an assumption that this is gzipped, then you must have a particular file in mind. If so, I would suggest making an entry in `resources/phenotypes.py` like one of the other resources and read it from there rather than taking it as an input arg